### PR TITLE
fix(scaling): Revert scaling changes

### PR DESCRIPTION
**Pull request by madhavilosetty-intel**

### CODE REVIEW

File: .mpsrc [hunk 1]
```diff
@@ -30,11 +30,11 @@
     "distributed_kv_name": "HashiCorpConsul",
```
Comment (brianosburn-intel) [inline .mpsrc:30]: are any of these being used anymore:     
"distributed_kv_ip": "localhost",
    "distributed_kv_port": "8500",
    "distributed_kv_name": "HashiCorpConsul",
    "web_proxy_port": "8100",
    "network_adaptor": "eth0",

File: src/utils/constants.ts [hunk 2]
```diff
@@ -57,10 +57,3 @@ export enum KeyValueStore {
   Unknown = 'Unknown',
```
Comment (brianosburn-intel) [inline src/utils/constants.ts:57]: is this being used anymore?

File: test/db.test.ts [hunk 3]
```diff
@@ -35,7 +35,6 @@ describe('Use GUID allowlisting: ', () => {
     distributed_kv_name: "",
```
Comment (brianosburn-intel) [inline test/db.test.ts:35]: remove unused scaling variables

File: test/db.test.ts [hunk 4]
```diff
@@ -134,7 +134,6 @@ describe('Do not use GUID allowlisting: ', () => {
...
```
Comment (brianosburn-intel) [inline test/db.test.ts:134]: please remove if not being used anymore

**Review by brianosburn-intel — APPROVED** [review]

File: .mpsrc (hunk 1 above)
Comment (madhavilosetty-intel) [reply .mpsrc]: done

File: test/db.test.ts (hunk 3 above)
Comment (madhavilosetty-intel) [reply test/db.test.ts]: done

File: test/db.test.ts (hunk 4 above)
Comment (madhavilosetty-intel) [reply test/db.test.ts]: done

**Review by rsdmike — APPROVED** [review]

**Pull request merged**